### ISSUE TITLE
Add support for Python 3

### ISSUE THREAD

**Issue by reinhardt** [collective/collective.honeypot#1]

**Comment by reinhardt** [comment]
I only just noticed that the Plone 5 support is not finished yet. That should of course be done before adding Python 3 support.

**Comment by tisto** [comment]
@reinhardt @mauritsvanrees we would be interested in a Plone 5.3/Python 3 compatible release as well. Our use case is PFG, not sure if that is already covered. Anything we can help with? cc @rodfersou

**Comment by mauritsvanrees** [comment]
I am pretty sure this package needs various changes for even Plone 5.0. We had it in the zestsoftware.nl site for a while on 5.0, but I removed it there because I knew things were either missing or even breaking (not sure what). Or I *thought* I had removed it, but it seems it is still there.
I only have it on only one other site, on Plone 4.3.

If someone wants to work on this, I would say:
- Drop Plone 4 support, so create a major breaking release.
- Focus on support for Plone 5.2 on Py 2.7 and 3.
- Plone 5.1 is nice to have, but don't spend hours on it.
- Forget about Plone 5.0 (though I guess if 5.1 works, 5.0 may work as well).

I don't think I have time to properly review big pull requests, though if if have a chance, I will try. I can grant PyPI access where needed.

**Comment by mauritsvanrees** [comment]
I have released 1.0.3 with the fix from @rodfersou and made the collective user owner on PyPI.

**Comment by mauritsvanrees** [comment]
I have created a branch 1.x and pushed it. Currently the same as master.
1.x would be for Plone 4 support (and Plone 3 it seems.)

I have reactivated Travis: https://travis-ci.org/github/collective/collective.honeypot/branches
Last build on master was five years ago and it was green. No doubt it will be red when we run it now.

Apparently I created a plone5 branch four years ago, but it [failed on Travis](https://travis-ci.org/github/collective/collective.honeypot/jobs/136749078), for starters because various templates or views no longer exist in Plone 5.

I am not sure if the plone5 branch is worth starting from, but I guess it is at least worth a look. Well, apparently I did only two commits there, of which the [last one](https://github.com/collective/collective.honeypot/commit/310ff62bea58e89cf2b3d618a7f0309a319022d7) is the most interesting. Anyone working on this should definitely look at that one. But feel free to start on a different branch.
